docs/UserGuide.md

```diff
@@ -18,6 +21,19 @@ HR manager's easy to use features will help you to arrange for upcoming intervie
 The data you have provided will also be stored safely and securely for subsequent uses,
 transferable to other devices too!
 
+### Purpose
+The purpose of this user guide is to give a complete documentation of HR Manger and allow you to easily get started on HR Manager.
+The user guide is also the easiest way for you to troubleshoot any issues and clarify any questions you have with HR Manager.<br>
+Lets simplify your HR processes!
+
+### How to use this user guide?
+This user guide is a complete guide for all commands and features for HR Manager.
+You can read the whole guide for a comprehensive understanding of the application.
+<br><br>
+Alternatively, you can quickly get started using the [Quick Start](#Quick start) and [Command Summary](#Command summary).<br>
```

Comment:
Not sure why but the links do not work.
![image](https://user-images.githubusercontent.com/61644924/140603016-24d9ec3e-f167-4090-8a78-cb0b2f7578c4.png)